.gitignore

```diff
@@ -76,6 +76,7 @@ build/Release
 # Dependency directories
 node_modules
 jspm_packages/
+package-lock.json
```

Comment:
Generally we'd want to commit these unless it's existence is causing issues (Which should be rare)
```suggestion

```